hardhat.config.ts

```diff
@@ -12,6 +12,15 @@ import {
     defaultLocalhostNetworkParams,
 } from 'hardhat/internal/core/config/default-config';
 
+const managerAccount =
+    Number(process.env.IEXEC_VOUCHER_MANAGER_ACCOUNT_INDEX) ||
```

Comment:
In named accounts you can specify, the address of the wallet as a string or specify an index of the wallet that is inferred from the used mnemonic. In integration tests, we specify indexes because we use Hardhat default mnemonic. In deployment we will certainly use the address because wallets are going to be separate from each other.

Reply:
I don't think we'll use bellecour field for now, we might c13 field with hardcoded value

Reply:
I have removed Bellecour related config to make the context of this PR clearer. It will be added later, properly.